.github/workflows/tier-1.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: Tier 1 jobs
```

Comment:
I think we should leave the title as
`1 - Pull Requests` rather than `Tier 1 jobs`
Because Tier 1 will always be related to pull requests. Tier 1 jobs is not really meaningful

Reply:
Good catch. Agree 100%. I will adjust it